demisto_sdk/commands/content_graph/objects/base_content.py

```diff
@@ -180,7 +181,12 @@ class BaseContent(BaseNode):
     git_status: Optional[GitStatuses]
     old_base_content_object: Optional["BaseContent"] = None
 
-    def _save(self, path: Path, data: dict):
+    def _save(
+        self,
+        path: Path,
+        data: dict,
+        predefined_keys_to_keep: Optional[Tuple[str]] = None,
```

Comment:
can you explain?
and add a docstring to the method

Reply:
When we want to save the info after the fixes, we take all the non-empty class vars (which translate into fields in the dict) and save them.
However, there are several fields that are a must as a part of the content item's structure, so I added these predefined fields to ensure they'll be added even if they're empty.